doc/supported_file_formats.adoc

```diff
@@ -35,12 +57,41 @@ Minetest's support of various model formats can be seen as lacking. This is a si
 
 [grid="all"]
 [options="header",stripes=even,cols="^.^,<.^,^.^,^.^,^.^,^.^,^.^,^.^"]
-|=======================
-|Format ^|General Usecase | File Format | Are static meshes supported? | Are animated meshes supported? | Implementation limitations | Blender importer | Blender exporter
-| OBJ (`.obj`) | Static meshes | Text | Yes | No | `.mtl` material files are not supported. 2+| Built-in
-| Blitz3D model (`.b3d`) | Animated meshes | Binary | Yes ^[1]^ | Yes | N/A .2+| None | https://github.com/GreenXenith/io_scene_b3d[io_scene_b3d]
-| DirectX 2.0 model (`.x`) | Animated meshes in text format | Binary/Text | Yes ^[1]^ | Yes | N/A | https://github.com/minetest/io_scene_x[io_scene_x] (with caveats; see README.md)
-|=======================
+|===
+| Format
+^| General Usecase
+| File Format
+| Are static meshes supported?
+| Are animated meshes supported?
+| Implementation limitations
+| Blender importer
+| Blender exporter
+
+| OBJ (`.obj`)
+| Static meshes
+| Text
+| Yes
+| No
+| `.mtl` material files are not supported.
+2+| Built-in
+
+| Blitz3D model (`.b3d`)
+| Animated meshes
+| Binary
+| Yes ^[1]^
+| Yes
+| N/A
+.2+| None
+| https://github.com/GreenXenith/io_scene_b3d[io_scene_b3d]
+
+| DirectX 2.0 model (`.x`)
+| Animated meshes in text format
+| Binary/Text
+| Yes ^[1]^
+| Yes
+| N/A
+| https://github.com/minetest/io_scene_x[io_scene_x] (with caveats; see README.md)
+|===
 
 ^[1]^ Not recommended since it will have the increased performance cost of an animated mesh.
```

Comment:
Citation needed.

Reply:
Hecks claims that Minetest creates AnimatedMeshSceneNode objects for all B3D and X models even if they are static without any animation ([link](https://wiki.minetest.net/Using_Blender#File_format_support)), although... from looking at the code now it doesn't look like this really is the case ([src/client/content_cao.cpp](https://github.com/minetest/minetest/blob/master/src/client/content_cao.cpp#L763)). I'll remove it for now until there is some further evidence this is the case.